_includes/api/en/5x/res-render.md

```diff
@@ -8,7 +8,7 @@ Optional parameters:
 
 The `view` argument is a string that is the file path of the view file to render. This can be an absolute path, or a path relative to the `views` setting. If the path does not contain a file extension, then the `view engine` setting determines the file extension. If the path does contain a file extension, then Express will load the module for the specified template engine (via `require()`) and render it using the loaded module's `__express` function.
 
-For more information, see [Using template engines with Express](/guide/using-template-engines.html).
+For more information, see [Using template engines with Express](/en/guide/using-template-engines.html).
```

Comment:
```suggestion
For more information, see [Using template engines with Express](/{{page.lang}}/guide/using-template-engines.html).
```

I will suggest to avoid hardcode the language and use `{{page.lang}}`. @zareefhasan here you can find another PR as a reference https://github.com/expressjs/expressjs.com/pull/1418/files 👍